fpm/recipes/rubygem-hiera-eyaml-gpg/recipe.rb

```diff
@@ -1,4 +1,24 @@
-class RubyHieraEyamlGPG < FPM::Cookery::RubyGemRecipe
-  name    "hiera-eyaml-gpg"
-  version "0.6"
+class RubyHieraEyamlGpg < FPM::Cookery::Recipe
+  # Do not use the RubyGemRecipe class as that will not cope with
+  # environments running rbenv. Instead write a custom recipe that will
+  # install ruby gems in standard ruby_vendors directory.
+  name    'rubygem-hiera-eyaml-gpg'
+  version '0.4'
+  homepage 'https://github.com/sihil/hiera-eyaml-gpg'
+
+  source "https://github.com/sihil/hiera-eyaml-gpg/archive/v#{version}.tar.gz"
+  sha256 "bd1d3fb2197af14420efa9e598e99d6fc9b659dd429e58c70ea7c457b742dd16"
+
+  maintainer 'GOV.UK <govuk-dev@digital.cabinet-office.gov.uk>'
+
+  depends 'hiera-eyaml (>=1.3.8)', 'ruby-gpgme (>=2.0.0)'
```

Comment:
We don't have a `hiera-eyaml` package installed or available on our Puppetmaster so I'm a bit confused about whether this `depends` line can ever be satisfied.